src/OrchardCore.Modules/OrchardCore.AuditTrail/package.json

```diff
@@ -5,29 +5,29 @@
   "license": "MIT",
   "description": "A diff module for OrchardCore.",
   "dependencies": {
-    "bootstrap": "4.6.1",
+    "bootstrap": "5.1.3",
```

Comment:
Update to bs5 ?

Reply:
That's because we will merge the bootstrap 5 admin PR this week.